fiboa_cli/datasets/data-files/us_fips.csv

```diff
@@ -0,0 +1,52 @@
+fips:name
```

Comment:
```suggestion
fips,name
```

Was that meant to be a comma? If not, it only has one column header...

Reply:
Ah thanks. I changed the ";" separator to "," in order to make it a normal csv